release/src/module_dependencies/main.py

```diff
@@ -86,7 +86,7 @@ def getVersion(balModule):
     version = ''
     for line in data:
         processedLine = line.decode("utf-8")
-        if 'version' in processedLine:
+        if re.match('version=', processedLine):
```

Comment:
This would match anything with `version=` right?

What if there's an entry with `someversion=x.x.x`?

```suggestion
        if re.match('^\s+version=', processedLine):
```

Reply:
This only identifies strings starting with `version=` 
`someversion=` will not be matched

Reply:
It will. That's the way Regex works right? `someVersion=` will not be matched, but `someversion=` will. Isn't it?

Reply:
Python [match()](https://docs.python.org/3/library/re.html#re.Pattern.match) function only matches if the string starts with the given pattern